### ISSUE TITLE
Canary Checker Image check details modal returns 401

### ISSUE THREAD

**Issue by PrestonCS** [flanksource/flanksource-ui#2275]
The main page now loads but getting details of a check still gets a 401

![image](https://github.com/user-attachments/assets/98cb8208-b76b-4942-9b3f-a4c27e215871)

_Originally posted by @PrestonCS in https://github.com/flanksource/flanksource-ui/issues/2267#issuecomment-2344020081_

**Comment by moshloop** [comment]
@PrestonCS this is fixed in the the most recent betas

**Comment by PrestonCS** [comment]
@moshloop How can I get a hold of a beta release? Or when will this be added to main releases?

**Comment by moshloop** [comment]
@PrestonCS You can use version [v1.0.260-beta.239](https://github.com/flanksource/canary-checker/releases/tag/v1.0.260-beta.239)

**Comment by PrestonCS** [comment]
Apparently some settings changed in the beta 

```
14:03:59.758 INF Watching /app/canary-checker.properties for changes
14:03:59.758 INF Loading properties from /app/canary-checker.properties
Error: unknown flag: --cache-timeout
Usage:
  canary-checker operator [flags]

Flags:
      --agent-name string               name of this agent
      --artifact-connection string      Specify the default connection to use for artifacts
      --db string                       Connection string for the postgres database (default "DB_URL")
      --db-log-level string             Set gorm logging level. trace, debug & info (default "error")
      --db-metrics                      Expose db metrics
      --db-migrations                   Run database migrations
      --db-schema string                Postgres schema (default "public")
      --default-window string           Default search window (default "1h")
      --disable-kubernetes              Disable Kubernetes integration
      --disable-postgrest               Disable PostgREST. Deprecated (Use --postgrest-uri '' to disable PostgREST)
      --enable-leader-election          Enabling this will ensure there is only one active controller manager
      --executor                        If false, only serve the UI and sync the configs (default true)
  -h, --help                            help for operator
      --httpPort int                    Port to expose a health dashboard  (default 8080)
      --include-canary strings          Only run canaries matching the given names
      --include-check strings           (Deprecated: use --include-canary) Run matching canaries - useful for debugging
      --include-labels strings          Only run canaries matching the given label selector
      --include-namespace strings       a comma separated list of namespaces whose canary should be run
      --k8s-log-level int               Kubernetes controller log level (default -1)
      --name string                     Server name shown in aggregate dashboard (default "local")
  -n, --namespace string                Watch only specified namespace, otherwise watch all
      --postgrest-anon-role string      PostgREST role for unauthenticated connections (default "postgrest_anon")
      --postgrest-jwt-secret string     JWT Secret Token for PostgREST (default "PGRST_JWT_SECRET")
      --postgrest-log-level string      PostgREST log level (default "info")
      --postgrest-max-rows int          A hard limit to the number of rows PostgREST will fetch (default 2000)
      --postgrest-role string           PostgREST role for authentication connections (default "postgrest_api")
      --postgrest-uri string            URL for the PostgREST instance to use. If localhost is supplied, a PostgREST instance will be started (default "http://localhost:3000")
      --public-endpoint string          Host on which the health dashboard is exposed. Could be used for generting-links, redirects etc. (default "http://localhost:8080")
      --upstream-host string            central canary checker instance to push/pull canaries
      --upstream-insecure-skip-verify   Skip TLS verification on the upstream servers certificate
      --upstream-max-age duration       upstream reconciliation max age (default 48h0m0s)
      --upstream-page-size int          upstream reconciliation page size (default 500)
      --upstream-password string        upstream password
      --upstream-user string            upstream username
      --webhookPort int                 Port for webhooks  (default 8082)

Global Flags:
      --color                       Print logs using color (default true)
      --json-logs                   Print logs in json format to stderr
      --log-fail                    Log every failing check
      --log-level string            Set the default log level (default "info")
      --log-pass                    Log every passing check
      --log-to-stderr               Log to stderr instead of stdout
  -v, --loglevel count              Increase logging level
      --otel-collector-url string   OpenTelemetry gRPC Collector URL in host:port format
      --otel-service-name string    OpenTelemetry service name for the resource (default "canary-checker")
      --prometheus string           URL of the prometheus server that is scraping this instance
      --report-caller               Report log caller info

version: v1.0.260-beta.239
 
Stream closed EOF for monitoring/canary-checker-54fd48b758-fmbjc (canary-checker)
```
